Plan/common/src/main/resources/assets/plan/web/network.html

```diff
@@ -557,32 +567,49 @@ <h6 class="m-0 font-weight-bold col-black"><i
                                 </table>
                             </div>
                         </div> <!-- Trends for 30 days-->
+
                         <!-- Insights -->
-                        <div class="col-lg-4 mb-4 col-sm-12">
-                            <div class="card shadow mb-4">
-                                <div class="card-header py-3">
-                                    <h6 class="m-0 font-weight-bold col-black"><i
-                                            class="far fa-fw fa-life-ring col-red"></i>
-                                        Insights for 30 days</h6>
+                        <div class="col-lg-4 col-sm-12">
+                            <div class="mb-4 col-lg-12 col-sm-12">
```

Comment:
It is best not to add a nested column here (might lead to some issues on smaller screens), instead adding the new pie as a card
```suggestion
```